jsettlers.logic/src/jsettlers/logic/movable/Movable.java

```diff
@@ -78,7 +78,7 @@
 
 	private ShortPoint2D position;
 
-	private ShortPoint2D moveToRequest = null;
+	private ShortPoint2D targetPosition = null;
```

Comment:
so targetPosition ok, isnt it? As its not some sort of  "Request Object" I think this might be better.